Resources/Audio/Effects/license.txt

```diff
@@ -9,3 +9,4 @@ adminhelp.ogg taken from https://github.com/tgstation/tgstation/blob/d775e1ac804
  Actual source is https://freesound.org/people/martian/sounds/19261/ (CC0)
  The sound had been reversed and the volume altered.
 
+voteding.ogg taken from "Bike, Bell Ding, Single, 01-01.wav" by InspectorJ (www.jshaw.co.uk) of Freesound.org at https://freesound.org/people/InspectorJ/sounds/484344/
```

Comment:
Probably nice to also write the exact license behind it, in case freesound.org ever dies or whatever (CC BY 3.0)

Reply:
sure okay